lib/solr/query/request/boosting/geodist_function.rb

```diff
@@ -17,7 +17,7 @@ def initialize(field:, latitude:, longitude:)
             freeze
           end
 
-          def to_solr_s
+          def to_solr_s(core:)
```

Comment:
this is wrong too, core shouldn't be required to convert something to a solr syntax